spark/dl/src/test/scala/com/intel/analytics/bigdl/keras/nn/MergeSpec.scala

```diff
@@ -36,6 +36,20 @@ class MergeSpec extends KerasBaseSpec {
     seq.forward(input) should be (input1 + input2)
   }
 
+  "merge method" should "work correctly" in {
```

Comment:
Let's refer to the python and add more tests here. i.e nested cases, validate the grad and output etc...

Reply:
or put them into `test_newapi.py`  if it's hard to get the std and running_mean from scala side.

Reply:
Tested on Python side.